_site/Software/SnPM13/index.html

```diff
@@ -103,7 +103,7 @@ <h2><a name="news">Introduction</a> </h2>
    </dt> 
    <dd>
     Citation of the SnPM software can be made with reference to this URL 
-    <a href="http://nisox.org/Software/SnPM13/"><tt>http://nisox.org/Software/SnPM13/</tt></a>; please also note the version (i.e. SnPM13) and the date that you last checked for updates. Concepts implemented in the SnPM software are best described in the Nichols &amp; Holmes (2001) paper; see 
+    <a href="http://nisox.org/Software/SnPM13/"><tt>http://nisox.org/Software/SnPM13/</tt></a>; please also note the version (i.e. SnPM13) in any citation. Concepts implemented in the SnPM software are best described in the Nichols &amp; Holmes (2001) paper; see 
```

Comment:
Here, and all recommendations for Citation (in other files) can we issue better best practice... i.e. give the *exact* version.  Something like:

> please also note the exact version (e.g. SnPM13.1.06) to maximise reproducibility.  The full version number can be found from the `snpm('ver')` command.

(check that `snpm('ver')`  works!)